lib/core/Head.js

```diff
@@ -30,9 +31,13 @@ class Head extends React.Component {
         }/styles/${highlight.theme}.min.css`;
 
     // ensure the siteUrl variable ends with a single slash
-    const siteUrl =
-      (this.props.config.url + this.props.config.baseUrl).replace(/\/+$/, '') +
-      '/';
+    const siteUrl = urljoin(
+      urljoin(this.props.config.url, this.props.config.baseUrl).replace(
```

Comment:
We can do better with
```js
urljoin(this.props.config.url, this.props.config.baseUrl)
```

This is because we are using `url-join` & don't need to handle trailing slash ourself